<launch>
  <arg name="camera_id" default="0"/>
  <arg name="camera_name" default="left"/>
  <arg name="camera_info_url" default=""/>

  <node pkg="usbcam" type="usbcam_node" output="screen" name="$(arg camera_name)" >
    <param name="camera_id" type="int" value="$(arg camera_id)"/>
    <param name="camera_name" type="str" value="$(arg camera_name)"/>
    <param name="camera_info_url" type="str" value="$(arg camera_info_url)"/>
  </node>
</launch>
